### DISCUSSION TITLE
Problems running after upgrading to net6.0

### DISCUSSION THREAD

**Discussion by craftyweb** [OrchardCMS/OrchardCore#11365] [Q&A] [answered]
Hello,

I've upgraded my OrchardCore Solution to the latest version that only supports net6.0. After some small changes I was able to build my solution.

When I try to run the application with IIS I get the following errors. Does anyone have an idea how to solve this? 

![screencapture-localhost-schriecksfriethuysnew-be-2022-03-12-12_03_08](https://user-images.githubusercontent.com/61787638/158015514-ad46a924-0aa9-4b96-94ba-a35bd45df6ed.png)

**Comment by craftyweb** [comment, marked as answer]
Had some problems in my csproj file of OrchardCore.Cms.Web. replaced it with the propertygroup of the csproj file in the OrchardCore github repository and now it's fixed

**Comment by MikeAlhayek** [comment]
The real solution to this problem is to look at the `.csproj` file of the host project and remove any or both of the following lines

```
<RazorRuntimeCompilation>true</RazorRuntimeCompilation>
<PreserveCompilationReferences>true</PreserveCompilationReferences>
```
